source/common/util/counter.ts

```diff
@@ -68,8 +79,8 @@ function getCleanedWords (ast: ASTNode, from = 0, to?: number): string[] {
  *
  * @return  {number}         The number of words in the file.
  */
-export function countWords (ast: ASTNode, from = 0, to?: number): number {
-  return getCleanedWords(ast, from, to).length
+export function countWords (ast: ASTNode, locale: string | undefined, from = 0, to?: number): number {
```

Comment:
```suggestion
export function countWords (ast: ASTNode, locale?: string, from = 0, to?: number): number {
```

We might want to rethink where we pass in `from`. I'm almost inclined to change the argument order to `ast, from, locale, to`, but I'm open to suggestions.